README.md

```diff
@@ -58,6 +58,44 @@ default unless you set the `QT_API` environment variable.
 * `pyside6` (to use PySide6).
 
 
+### Methods, helpers and QtPy namespace specifics
+
+As part of providing compatibility for the different Python Qt bindings available, QtPy does some modules mapping and exposes a `compat` module. Bellow you can check some elements to take into account when using QtPy.
+
+#### Module aliases and constants
+
+* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are replaced by `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property` following the Qt5 module layout.
```

Comment:
```suggestion
* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are aliased to `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property` following the Qt5 module layout.
```

Is this more accurate?

Reply:
Well I used replaced since these elements are not available from QtPy (you can't do `from qtpy.QtCore import pyqtSignal`). So they are not just aliased but also deleted. Maybe there is a word to describe that and that is more clear than `replaced`? Maybe something like `...(available on PyQt5/6) are deleted and aliased to...`

Reply:
Okay, thanks for the clarification—I wasn't sure which you intended to mean. "Replaced" is okay (deleted implies that a destructive modification is made on the original packages, rather than simply not exposed by QtPy), but to be more clear, you could say

```suggestion
* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are instead exposed as `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property`, respectively, following the Qt5 module layout.
```

What about PySide?

Reply:
Ok :+1: Regarding PySide, it uses `Signal`, `Slot` and `Property` already so QtPy doesn't need to do anything